### PR TITLE
Add FPS patch for Modern Warfare 2 (TU6)

### PR DESCRIPTION
**Pull request by Pirriu**

### CODE REVIEW

**Comment by AdrianCassar** [comment]
You can set dvars for TU 6 and below via the command in the config `cl = "+set com_maxfps 120"`

File: patches/41560817 - Modern Warfare 2 (TU6).patch.toml [hunk 1]
```diff
@@ -1,8 +1,19 @@
...
 
+[[patch]]
+    name = "120 FPS"
+    desc = "Set Frame Rate Limit to 120 FPS"
+    author = "iTzZ Fenix"
+    is_enabled = false
+
+    [[patch.be8]]
+        address = 0x82280588 # com_maxfps
+        value = 0x78
```
[the remark this answers is not in the document]
Comment (Pirriu) [reply patches/41560817 - Modern Warfare 2 (TU6).patch.toml:15]: Yes, but more than 120FPS may start breaking some fisics.

File: patches/41560817 - Modern Warfare 2 (TU6).patch.toml [hunk 2]
```diff
@@ -1,8 +1,19 @@
 title_name = "Call of Duty: Modern Warfare 2"
 title_id = "41560817" # AV-2071
-hash = "A3FFB7C11B688FE5" # default_mp.xex
+hash = "A3FFB7C11B688FE5" # default_mp.xex TU6 not modded
+#hash = "52276FB138B533AF" # default_mp.xex TU6 RSA already removed
 #media_id = "2C8C0267" # Disc (USA, Europe): http://redump.org/disc/11809
 
+[[patch]]
+    name = "120 FPS"
+    desc = "Set Frame Rate Limit to 120 FPS"
```
Comment (Margen67) [inline patches/41560817 - Modern Warfare 2 (TU6).patch.toml:9]:
```suggestion
    desc = "Sets frame rate limit to 120 FPS. Anything higher breaks certain physics interactions."
```

File: patches/41560817 - Modern Warfare 2 (TU6).patch.toml [hunk 3]
```diff
@@ -1,8 +1,19 @@
...
+[[patch]]
+    name = "120 FPS"
+    desc = "Sets frame rate limit to 120 FPS. Anything higher may break some physics interactions."
+    author = "iTzZ Fenix"
+    is_enabled = false
+
+    [[patch.be8]]
+        address = 0x82280588 # com_maxfps
+        value = 0x78
+
```
Comment (Margen67) [inline patches/41560817 - Modern Warfare 2 (TU6).patch.toml:16]:
```suggestion

```

File: patches/41560817 - Modern Warfare 2 (TU6).patch.toml [hunk 4]
```diff
@@ -1,8 +1,18 @@
...
 
+[[patch]]
+    name = "120 FPS"
+    desc = "Sets frame rate limit to 120 FPS. Anything higher may break some physics interactions."
+    author = "iTzZ Fenix"
+    is_enabled = false
+
+    [[patch.be8]]
+        address = 0x82280588 # com_maxfps
+        value = 0x78
```
Comment (Margen67) [inline patches/41560817 - Modern Warfare 2 (TU6).patch.toml:15]:
```suggestion
        value = 0x78

```

File: patches/41560817 - Modern Warfare 2 (TU6).patch.toml [hunk 5]
```diff
@@ -1,8 +1,19 @@
 title_name = "Call of Duty: Modern Warfare 2"
 title_id = "41560817" # AV-2071
-hash = "A3FFB7C11B688FE5" # default_mp.xex
+hash = "A3FFB7C11B688FE5" # default_mp.xex TU6 not modded
+#hash = "52276FB138B533AF" # default_mp.xex TU6 RSA already removed
```
Comment (Margen67) [inline patches/41560817 - Modern Warfare 2 (TU6).patch.toml:4]:
```suggestion
hash = [
    "A3FFB7C11B688FE5" # default_mp.xex
    #"52276FB138B533AF" # default_mp.xex, RSA checks removed
]
```

**Pull request merged**